lms/static/js/student_profile/views/learner_profile_fields.js

```diff
@@ -16,22 +17,29 @@
             },
 
             showNotificationMessage: function () {
-                var accountSettingsLink = '<a href="' + this.options.accountSettingsPageUrl + '">' + gettext('Account Settings page.') + '</a>';
+                var accountSettingsLink = StringUtils.interpolate('<a href="{url}">Account Settings page.</a>',
```

Comment:
Unfortunately this is one of the files that I started to convert but didn't finish. You can see what I did here:

https://github.com/edx/edx-platform/pull/11856/files

What I settled on was this:

                    accountSettingsLinkHtml = HtmlUtils.joinHtml(
                        HtmlUtils.interpolateHtml('<a href="{settings_url}">', {settings_url: accountSettingsPageUrl}),
                        gettext('Account Settings page.'),
                        HtmlUtils.HTML('</a>')
                    );